pan/types.pan

```diff
@@ -1,53 +1,50 @@
-################################################################################
-# This is 'namespaces/standard/pan/types.tpl', a pan-templates's file
-################################################################################
-#
-# VERSION:    3.2.7, 21/08/09 22:22
-# AUTHOR:     Martin Bock
-# MAINTAINER: Marco Emilio Poleggi <Marco.Emilio.Poleggi@cern.ch>, German Cancio <German.Cancio.Melia@cern.ch>, Michel Jouvin <jouvin@lal.in2p3.fr>
-# LICENSE:    http://cern.ch/eu-datagrid/license.html
-#
-################################################################################
-# Coding style: emulate <TAB> characters with 4 spaces, thanks!
-################################################################################
-#
-# Data type and function definitions for the following basic types:
-#
-#     * asndate
-#     * isodate
-#     * hwaddr
-#     * ip (ipv4 and ipv6)
-#     * fqdn
-#     * hostname
-#     * shorthostname
-#     * port
-#     * hostport
-#     * URI
-#     * absoluteURI
-#     * hostURI
-#     * email
-#     * network_name
-#
-################################################################################
+@documentation{
+This is 'namespaces/standard/pan/types.tpl', a pan-templates's file
+
+VERSION:    3.2.7, 21/08/09 22:22
+AUTHOR:     Martin Bock
+MAINTAINER: Marco Emilio Poleggi <Marco.Emilio.Poleggi@cern.ch>, German Cancio <German.Cancio.Melia@cern.ch>, Michel Jouvin <jouvin@lal.in2p3.fr>
```

Comment:
I think so, I was just wondering about that as well.. I will include a `@maintainer` block for each one